cli/command_snapshot_create.go

```diff
@@ -118,6 +119,7 @@ func (c *commandSnapshotCreate) run(ctx context.Context, rep repo.RepositoryWrit
 		return errors.New("description too long")
 	}
 
+	//nolint:contextcheck
```

Comment:
Why do we need this nolint directive here? it seems to be disabling the check for a non-inherited context, but where do we use it?